app/models/processed_submission_metric.rb

```diff
@@ -0,0 +1,5 @@
+class ProcessedSubmissionMetric < ApplicationRecord
+  def self.total
+    first.&total || create(total: 0)
```

Comment:
Could we put this creation of the first object in the migration? It will be the only time its called, seems overkill to have it in the application code.

Would probably require a `def up` / `def down` in the migration to add/remove it for rollbacks.

Reply:
Actually it looks like `increment` does this for us? https://apidock.com/rails/ActiveRecord/Persistence/increment